From: Seth Heasley <seth.heasley@intel.com>
Subject: Add Patsburg PCH support to SLES11 SP1
References: bnc#650545, bnc#663513, FATE#311346, FATE#311838
Patch-mainline: 2.6.36, 2.6.37, 2.6.38

Acked-by: Jean Delvare <jdelvare@suse.de>
Automatically created from "patches.drivers/intel-patsburg-support.patch" by xen-port-patches.py

--- sle11sp2-2011-05-04.orig/arch/x86/pci/irq-xen.c	2011-05-05 15:45:24.000000000 +0200
+++ sle11sp2-2011-05-04/arch/x86/pci/irq-xen.c	2011-05-05 16:10:08.000000000 +0200
@@ -598,8 +598,8 @@ static __init int intel_router_probe(str
 	case PCI_DEVICE_ID_INTEL_ICH10_3:
 	case PCI_DEVICE_ID_INTEL_CPT_LPC1:
 	case PCI_DEVICE_ID_INTEL_CPT_LPC2:
-	case PCI_DEVICE_ID_INTEL_PATSBURG_LPC_0:
 	case PCI_DEVICE_ID_INTEL_PATSBURG_LPC_1:
+	case PCI_DEVICE_ID_INTEL_PATSBURG_LPC_2:
 		r->name = "PIIX/ICH";
 		r->get = pirq_piix_get;
 		r->set = pirq_piix_set;
